### ISSUE TITLE
Product administration - Created On / Updated On read-only fields missing when editing a product

### ISSUE THREAD

**Issue by sumo300** [nopSolutions/nopCommerce#6320]
nopCommerce version: 4.50.3

After upgrading from 3.9 to 4.50.3, store owner noticed that the Created On and Updated On fields are no longer present when editing a product.  Requesting that these fields be brought back in a future version.

Steps to reproduce the problem: 

1. Use version 3.9
2. Edit a product
3. Observe at the bottom of the "General information" section, two read-only fields named Created On and Updated On.
4. Upgrade to 4.50.3
5. Edit a product
6. Observe, even in Advanced mode, there are no Created On / Updated On read-only fields visible

**Comment by AndreiMaz** [comment]
It's by design. These fields should be read-only. 

You have to customize the solution if you want to edit them

**Comment by sumo300** [comment]
I think you misunderstood the feature request. The fields don't display any more and the desire is to bring them back.  I was not asking for them to be editable. They existed as read-only in 3.9 and the store owner noticed they're missing in 4.50.3.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by sumo300** [comment]
@AndreiMaz I see nowhere in #1647 where it was actually discussed to completely remove those two read-only fields or why exactly.  I see in a commit it was just done, but one can only assume it was to provide some real estate for some of the other design changes.  Having those fields visible is an administrative user experience issue.  I'd request that you re-open to allow a discussion of where to put them or to truly leave it as-is, since they've been there for a long time and business customers come to expect they be present for purposes of inventory management for some types of businesses.

The use case is that in rare cases when a product is found via Google, but is no longer available, the business can inform the customer when it was removed (the Updated On date).

We've since turned off the setting that allows viewing the detail page of unpublished items.  Maybe this fully gets rid of the issue and there's no longer a use case.

Either way, I was hoping to turn this into an enhancement request rather than an answer of "by design" so that it could be discussed.
